back/libs/oidc-provider/src/services/oidc-provider-config-app.service.ts

```diff
@@ -6,26 +6,34 @@ import {
 
 import { Injectable } from '@nestjs/common';
 
+import { AppConfig } from '@fc/app';
 import { ConfigService } from '@fc/config';
-import { UserSession } from '@fc/core-fca';
+import { CoreFcaSession, UserSession } from '@fc/core-fca';
 import { throwException } from '@fc/exceptions/helpers';
 import { LoggerService } from '@fc/logger';
+import { OidcClientRoutes, OidcClientService } from '@fc/oidc-client';
 import { SessionService } from '@fc/session';
+import { TrackedEventContextInterface, TrackingService } from '@fc/tracking';
 
-import { OidcProviderRuntimeException } from '../exceptions';
-import { LogoutFormParamsInterface } from '../interfaces';
-import { OidcProviderErrorService } from './oidc-provider-error.service';
+import {
+  OidcProviderMissingAtHashException,
+  OidcProviderRuntimeException,
+} from '../exceptions';
+import { LogoutFormParamsInterface, OidcCtx } from '../interfaces';
+import { OidcProviderErrorService } from '../services';
```

Comment:
```suggestion
import { OidcProviderErrorService } from './oidc-provider-error.service';
```

The import from `services` creates a circular dependency, which results in NestJs being unable to load OidcProviderErrorService, resulting in the execution time error "Nest can't resolve dependencies of the OidcProviderConfigAppService".

I have found [this useful tool](https://manishbit97.medium.com/identifying-circular-dependencies-in-nestjs-using-madge-de137cd7f74f) to detect and help resolve circular dependencies problems.